terra-jupyter-base/Dockerfile

```diff
@@ -142,12 +135,11 @@ COPY custom $JUPYTER_HOME/custom
 COPY jupyter_notebook_config.py $JUPYTER_HOME
 
 # copy workspace_cromwell.py script and make it runnable by all users
+# TODO why not move this script into this repo?
 RUN curl -o /usr/local/bin/workspace_cromwell.py https://raw.githubusercontent.com/broadinstitute/cromwhelm/1ceedf89587cffd355f37401b179001f029f77ed/scripts/workspace_cromwell.py \
     && chmod +x /usr/local/bin/workspace_cromwell.py
 
 RUN chown -R $USER:users $JUPYTER_HOME \
-    # Disable nb_conda for now. Consider re-enable in the future
-    && jupyter nbextension disable nb_conda --py --sys-prefix \
```

Comment:
This line no longer works because the extension can't be found. No need to disable if it isn't there!